### PR TITLE
[WAF] Get started

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-10889.

### CODE REVIEW

File: content/waf/get-started.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+## Before you begin
+
+- Make sure that you have [set up a Cloudflare account](/fundamentals/setup/account/) and [added your domain](/fundamentals/setup/manage-domains/add-site/) to Cloudflare.
+
+- Users on the Free plan have access to the Cloudflare Free Managed Ruleset, a subset of the Cloudflare Managed Ruleset. The Free Managed Ruleset is deployed by default on Free plans and is not specifically covered in this guide.<br>If you are on a Free plan, you may skip to [5. Review traffic in security dashboards](#5-review-traffic-in-security-dashboards).
+
+## 1. Deploy the Cloudflare Managed Ruleset
+
+The [Cloudflare Managed Ruleset](/waf/managed-rules/reference/cloudflare-managed-ruleset/) protects against CVEs and known attack vectors. This ruleset is designed to identify common attacks using signatures, while generating low false positives. Rule changes are published on a weekly basis in the [WAF changelog](/waf/change-log/). Cloudflare may also add rules at any time during emergency releases for high profile zero-day protection.
```
Comment (marciocloudflare) [inline content/waf/get-started.md]:
```suggestion
The [Cloudflare Managed Ruleset](/waf/managed-rules/reference/cloudflare-managed-ruleset/) protects against Common Vulnerabilities and Exposures (CVEs) and known attack vectors. This ruleset is designed to identify common attacks using signatures, while generating low false positives. Rule changes are published on a weekly basis in the [WAF changelog](/waf/change-log/). Cloudflare may also add rules at any time during emergency releases for high profile zero-day protection.
```

File: content/waf/get-started.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    - **Expression**: `Bot Score less than 20 AND Verified Bot equals Off`
+    - **Action**: _Managed Challenge_
+
+For more information about the bot-related fields you can use in expressions, refer to [Bot Management variables](/bots/reference/bot-management-variables/).
+
+Once you have deployed the Cloudflare Managed Ruleset and a rule based on the attack score you will have achieved substantial protection, limiting the chance of false positives.
+
+## 4. (Optional) Deploy the Cloudflare OWASP Core Ruleset
+
+After configuring the Cloudflare Managed Ruleset and attack score, you can also deploy the [Cloudflare OWASP Core Ruleset](/waf/managed-rules/reference/owasp-core-ruleset/). This managed ruleset is Cloudflare's implementation of the OWASP ModSecurity Core Rule Set and its attack coverage significantly overlaps with Cloudflare Managed Ruleset by detecting common attack vectors such as SQLi and XSS.
```
Comment (marciocloudflare) [inline content/waf/get-started.md]:
```suggestion
After configuring the Cloudflare Managed Ruleset and attack score, you can also deploy the [Cloudflare OWASP Core Ruleset](/waf/managed-rules/reference/owasp-core-ruleset/). This managed ruleset is Cloudflare's implementation of the OWASP ModSecurity Core Rule Set. Its attack coverage significantly overlaps with Cloudflare Managed Ruleset by detecting common attack vectors such as SQLi and XSS.
```

Comment (marciocloudflare) [reply content/waf/get-started.md]: If I understood what you meant, I think adding a full stop improves legibility. Feel free to ignore tough

File: content/waf/get-started.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+{{</Aside>}}
+
+1. Go to your domain > **Security** > **WAF** and select the **Managed rules** tab.
+2. Under **Managed Rulesets**, select **Deploy** next to the Cloudflare OWASP Core Ruleset.<br>
+This will deploy the ruleset with the default configuration: paranoia level = _PL1_ and score threshold = _Medium - 40 and higher_.
+
+{{<details header="Ruleset configuration">}}
+
+Unlike the signature-based Cloudflare Managed Ruleset, the Cloudflare OWASP Core Ruleset is score-based. You select a certain paranoia level (levels vary from _PL1_ to _PL4_), which enables an increasing larger group of rules. You also select a score threshold, which decides when to perform the configured action. Low paranoia with a high score threshold usually leads to fewer false positives. For an example of how the OWASP Core Ruleset is evaluated, refer to [OWASP evaluation example](/waf/managed-rules/reference/owasp-core-ruleset/example/).
```
Comment (marciocloudflare) [inline content/waf/get-started.md]: PL1 is a lower level of paranoia than PL4? Or is it the other way around?

**Review by marciocloudflare — APPROVED** [review]

File: content/waf/get-started.md (hunk 3 above)
Comment (pedrosousa) [inline content/waf/get-started.md]:
```suggestion
Unlike the signature-based Cloudflare Managed Ruleset, the Cloudflare OWASP Core Ruleset is score-based. You select a certain paranoia level (levels vary from _PL1_ to _PL4_, where _PL1_ is the lowest level), which enables an increasing larger group of rules. You also select a score threshold, which decides when to perform the configured action. Low paranoia with a high score threshold usually leads to fewer false positives. For an example of how the OWASP Core Ruleset is evaluated, refer to [OWASP evaluation example](/waf/managed-rules/reference/owasp-core-ruleset/example/).
```

Comment (pedrosousa) [reply content/waf/get-started.md]: Clarified that PL1 is the lowest level.

**Pull request merged**